spec/tasks/translation_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require_relative '../../spec/support/tasks'  
+
+describe 'rake tasks' do
+  before do
+    fake_rails = double()
+    fake_rails.stub(:root) { Pathname.new('spec') }
+    stub_const("Rails", fake_rails)
+  end
+
+  describe 'translation:add_missing', type: :task do
+    let(:task) { Rake::Task["translation:add_missing"] }
+  
+    it 'add missing task is executed' do
```

Comment:
This should read like a sentence, i.e. "it is executed":

```suggestion
    it 'is executed' do
```